### PR TITLE
DESCRIPTION: fix some typos

### PR DESCRIPTION
**Pull request by florisvdh**

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Package: effectclass
 Type: Package
-Title: Classification and visulation of effects
+Title: Classification and visualization of effects
```
Comment (ThierryO) [inline DESCRIPTION]:
```suggestion
Title: Classification and visualisation of effects
```
I prefer the English version of English

**Review by ThierryO — APPROVED** [review]

File: DESCRIPTION (hunk 1 above)
Comment (florisvdh) [reply DESCRIPTION]: :+1: your choice  '...ise' is the US English version, '...ize' is UK :wink:

Comment (ThierryO) [reply DESCRIPTION]: Not according to the [Cambridge Dictionary](https://dictionary.cambridge.org/dictionary/english/visualize)

> visualize
verb [ T ] (UK usually visualise)

Comment (florisvdh) [reply DESCRIPTION]: Hmm... it seems you're absolutely right :hushed:  I've been taught Oxford English but been tricked, as it does not always follow British English...

> **Origin and recommendations**
The Oxford English Dictionary (OED) recommends -ize and notes that the -ise spelling is from French
>
> **Usage**
American spelling avoids -ise endings in words like organize, realize and recognize.
British spelling mostly uses -ise, while -ize is sometimes used

(From [Wikipedia](https://en.wikipedia.org/wiki/American_and_British_English_spelling_differences#-ise.2C_-ize_.28-isation.2C_-ization.29))

I'll need 30 days for a mental shift.
Or more probable, be confused forever.
Most probable of all: prefer good ol' Oxford English.

**Pull request merged**